README.md

```diff
@@ -37,7 +37,7 @@ $ pip install connect-sdk
 
 ### Example
 ```python
-from connect import FulfillmentAutomation
+from connect import FulfillmentAutomation, TierConfigRequestAutomation
```

Comment:
s/TierConfigRequestAutomation/TierConfigAutomation/

logic:
Fulfillment - TierConfig
FulfillmentRequest - TierConfigRequest
FulfillmentAutomation - TierConfigAutomation

Reply:
Right. It will be fixed in the next commit.